$NetBSD: patch-cc,v 1.1 2005/11/19 11:50:56 rillig Exp $

The SunPro compiler does not like non-static inline functions.

--- sieve/bc_emit.c.orig	Wed Oct 22 20:03:24 2003
+++ sieve/bc_emit.c	Sat Nov 19 12:48:06 2005
@@ -45,7 +45,7 @@ OF OR IN CONNECTION WITH THE USE OR PERF
 void dump(bytecode_info_t *d);
 #endif
 
-inline int write_int (int fd, int x)
+static inline int write_int (int fd, int x)
 {
     int y=htonl(x);
     return (write(fd, &y, sizeof(int)));
